clients/lsp-clangd.el

```diff
@@ -173,7 +173,8 @@ find a suitable one. Set this variable before loading lsp."
   "Clang default executable full path when found.
 This must be set only once after loading the clang client.")
 
-(defcustom lsp-clients-clangd-args '()
+(defcustom lsp-clients-clangd-args '("--header-insertion=never"
```

Comment:
While `--header-insertion-decorators` is indeed not working well with Emacs, the first one `--header-insertion` is actually a feature of `clangd`.
It works fine for most case so I think we shouldn't include `--header-insertion=never` here yet

Reply:
> It works fine for most case so I think we shouldn't include --header-insertion=never here yet

Good. I want to hear what others think about that as well and if there is no agreement I will revert that line and move it to FAQ section.

Reply:
I personally prefer `--header-insertion=never` because I think clangd's include insertion fixit is just as convenient and I don't want my editor to sneak in potentially expensive includes just because I autocompleted the wrong candidate by accident. However, I haven't had any issues with that option either so both ways are decent defaults IMHO

Reply:
I prefer to have automatic header insertion by default. JetBrains's CLion also has this feature enabled by default, for instance.

IMO, we should have the same default as the clangd VSCode extension, as it's the canonical extension developed by Clangd's team.